### ISSUE TITLE
Requesting a background highlight color for sentences during voiceover playback

### ISSUE THREAD

**Issue by Nik-09** [oppia/design-team#181]
**Platform**
 Design style guide

**Describe the request**
Highlighting sentences during voiceover playback should be implemented in a way that maintains the readability of the lesson. Currently, we highlight the entire lesson content during playback rather than individual sentences. However, applying the current highlight color ([#f3d140](https://github.com/oppia/oppia/blob/8b3b4cec44cd328ca92e02c106f8bdc3172f6bbe/core/templates/pages/exploration-player-page/learner-experience/tutor-card.component.css#L98)) to the background affects readability, as shown in the attached image. I would like to request the design team to provide a hexadecimal code for a lighter shade of yellow that enhances visibility without compromising readability.

![Image](https://github.com/user-attachments/assets/023182e4-3dc5-443f-93f5-a65ecfef9a8c)

**Context for the request**
In the project using [Azure for autogenerated voiceovers](https://docs.google.com/document/d/1f1IrsJyqYhjhHx-8IcT0a8OG6PvxsQR1X1BHvWhCWAo/edit?tab=t.0#bookmark=id.4cxr1rfa88yh), the requirement to highlight sentences during voiceover playback was raised.

**Link to PRD**
https://docs.google.com/document/d/1f1IrsJyqYhjhHx-8IcT0a8OG6PvxsQR1X1BHvWhCWAo/edit?tab=t.0#heading=h.liiqbmwmz8j8

**UXR success criteria**
For revisions of existing features, list the success criteria for the design here (i.e. what must the design achieve during UXR for it to be considered successful)? E.g. "When a user visits the topic page, they should be able to select their first lesson without needing further prompting."

**DESIGN APPROVAL CHECKLIST -- DO NOT EDIT.**

- [ ] UX Design approved (Android mocks should include dark mode)
- [ ] UX Writing approved
- [ ] Product approved

_For a design project to be completed, it needs approval from UXD, UXW and PM. Please tag the following reviewers when your project is ready (at least one reviewer for each of the bullet points below):_
- UXD: @rflore / @S4v8n (for Web projects), @seanlip (for Android projects)
- UXW: @davishedrick / @aileenyesand
- PM: @seanlip

**Comment by rflore** [comment]
Hi Nikhil, I was reviewing the color #FFD430 in Figma and used a color accessibility plug-in to review the color contrast and it passes the readability test. It's at a high contrast with yellow highlights(#FFD430) and black text (#1F1B08). I don't see any problem with using this color. Was there a reason why this was noted as not readable? 

Color Contrast Checker to match ([link](https://www.audioeye.com/color-contrast-checker/?utm_adgroup=WCAG-Check&hsa_acc=3628656526&hsa_cam=20558467141&hsa_grp=1319416404620278&hsa_ad=&hsa_src=s&hsa_tgt=kwd-82464729422020:loc-190&hsa_kw=wcag%20checker&hsa_mt=e&hsa_net=adwords&hsa_ver=3))
![Image](https://github.com/user-attachments/assets/4f114825-d558-4a4f-b5da-ec66987dc154)

**Comment by seanlip** [comment]
@rflore Thanks for taking a look! I think this is more related to aesthetics, actually. Do you think the yellow shade shown here is OK, or would a different colour be better?

**Comment by Nik-09** [comment]
> [@rflore](https://github.com/rflore) Thanks for taking a look! I think this is more related to aesthetics, actually. Do you think the yellow shade shown here is OK, or would a different colour be better?

I agree with @seanlip's comment. @rflore can you please share your thoughts on this?
Thank you!

**Comment by rflore** [comment]
> > [@rflore](https://github.com/rflore) Thanks for taking a look! I think this is more related to aesthetics, actually. Do you think the yellow shade shown here is OK, or would a different colour be better?
> 
> I agree with [@seanlip](https://github.com/seanlip)'s comment. [@rflore](https://github.com/rflore) can you please share your thoughts on this? Thank you!

Hi Nik, how does #F0FFD1 work as a highlight? This is more for our brand colors, but I'm curious if this blends with the blue background color from your screenshot. Can you please send a preview with this color so we can re-assess? Thanks!
